lib/groonga-query-log/command/run-regression-test.rb

```diff
@@ -252,6 +253,14 @@ def create_option_parser
                   "(#{@debug_rewrite})") do |boolean|
           @debug_rewrite = boolean
         end
+        parser.on("--execution-query-rate=RATE", Float,
+                  "You can specify execution query rate." +
+                  "The unit of this option is %." +
```

Comment:
If we use % for the unit, we must use `0..100` instead of `0..1`.

Reply:
I deleted "The unit of this option is %.".